espree.js

```diff
@@ -2595,49 +2616,56 @@ function parseObjectProperty() {
             )
         );
 
-    } else {
```

Comment:
oh yeah this used to be an `else` the current `harmony` prefers the early return style.  changed it during some debugging...